src/main/resources/db/migration/V0002__purge.sql

```diff
@@ -0,0 +1,259 @@
+DROP SEQUENCE IF EXISTS purge_job_audit_seq;
+DROP SEQUENCE IF EXISTS purge_job_audit_messages_seq;
+DROP TABLE IF EXISTS sdt_owner.purge_job_audit CASCADE;
+DROP TABLE IF EXISTS sdt_owner.purge_job_audit_messages CASCADE;
+
+CREATE TABLE sdt_owner.purge_job_audit
+(purge_job_id NUMERIC NOT NULL,
+job_start_date TIMESTAMP(6),
+job_end_date TIMESTAMP(6),
+retention_date TIMESTAMP(6),
+count_of_error_logs_deleted NUMERIC,
+count_of_individual_requests_deleted NUMERIC,
+count_of_bulk_submissions_deleted NUMERIC,
+count_of_service_requests_deleted NUMERIC,
+success NUMERIC);
+
+CREATE TABLE sdt_owner.purge_job_audit_messages
+(purge_job_message_id NUMERIC NOT NULL,
+purge_job_id NUMERIC,
+message_date TIMESTAMP(6),
+log_message VARCHAR(256));
+
+-- add indexes --
+CREATE INDEX pja_purge_job_id ON sdt_owner.purge_job_audit (purge_job_id);
+CREATE INDEX pjam_purge_job_message_id ON sdt_owner.purge_job_audit_messages (purge_job_message_id);
+
+CREATE UNIQUE INDEX purge_job_audit_pk ON sdt_owner.purge_job_audit (purge_job_id);
+CREATE UNIQUE INDEX purge_job_audit_messages_pk ON sdt_owner.purge_job_audit_messages (purge_job_message_id);
+
+-- add constraints --
+ALTER TABLE sdt_owner.purge_job_audit ADD CONSTRAINT purge_job_audit_pk PRIMARY KEY USING INDEX purge_job_audit_pk;
+ALTER TABLE sdt_owner.purge_job_audit_messages ADD CONSTRAINT purge_job_audit_messages_pk PRIMARY KEY USING INDEX purge_job_audit_messages_pk;
```

Comment:
Also need to set up a foreign key constraint between purge_job_audit_messages and purge_job_audit tables on the purge_job_id columns.  This will ensure that only rows with a corresponding entry in the purge_job_audit table can be inserted into purge_job_audit_messages table.

Reply:
Done